core/src/test/java/greencity/controller/EventsControllerTest.java

```diff
@@ -67,4 +76,163 @@ void getEventsCreatedByUser() {
 
     }
 
+    @Test
+    @SneakyThrows
+    void saveTest() {
+        ObjectMapper objectMapper = new ObjectMapper();
+        objectMapper.findAndRegisterModules();
+        String json = "{\n" +
+            "    \"title\":\"string\",\n" +
+            "    \"description\":\"stringstringstringstringstringstringstringstring\",\n" +
+            "    \"open\":true,\n" +
+            "    \"datesLocations\":[\n" +
+            "        {\n" +
+            "            \"startDate\":\"2023-05-27T15:00:00Z\",\n" +
+            "            \"finishDate\":\"2023-05-27T17:00:00Z\",\n" +
+            "            \"coordinates\":{\n" +
+            "                \"latitude\":1,\n" +
+            "                \"longitude\":1\n" +
+            "            },\n" +
+            "            \"onlineLink\":\"http://localhost:8080/swagger-ui.html#/events-controller\"\n" +
+            "        }\n" +
+            "    ],\n" +
+            "    \"tags\":[\"Social\"]\n" +
+            "}   ";
+
+        MockMultipartFile jsonFile =
+            new MockMultipartFile("addEventDtoRequest", "", "application/json", json.getBytes());
+
+        mockMvc.perform(multipart(EVENTS_CONTROLLER_LINK + "/create")
+            .file(jsonFile)
+            .principal(principal)
+            .accept(MediaType.APPLICATION_JSON)
+            .contentType(MediaType.APPLICATION_JSON))
+            .andExpect(status().isCreated());
```

Comment:
I see another response codes for `create` endpoint.

```
        @ApiResponse(code = 400, message = HttpStatuses.BAD_REQUEST),
        @ApiResponse(code = 401, message = HttpStatuses.UNAUTHORIZED)
```
        
Please add test for that cases too